packages/frontend/package.json

```diff
@@ -7,13 +7,13 @@
   "license": "Apache 2.0",
   "publisher": "SAP",
   "scripts": {
-    "ci": "npm-run-all clean build coverage",
+    "ci": "npm-run-all clean build coverage coverage:copy",
     "ci:artifacts_only": "npm-run-all clean build",
     "clean": "rimraf ./dist ./coverage",
     "build": "vue-cli-service build",
```

Comment:
You will need to sync with the change of vite.js migration as scripts has changed here... we no longer run the tests with `vue-cliservice`